bindings/matrix-sdk-ffi/src/api.udl

```diff
@@ -1,9 +1,19 @@
 namespace sdk {
+    void setup_tracing(LogLevel log_level);
+
     MediaSource media_source_from_url(string url);
     MessageEventContent message_event_content_from_markdown(string md);
     string gen_transaction_id();
 };
 
+enum LogLevel {
+    "Trace",
+    "Debug",
+    "Info",
+    "Warn",
+    "Error",
```

Comment:
Oh no, we are mixing spaces and tabs D:

Can you do four spaces in the whole file please?

Reply:
Strange, neither me nor sublime find anything wrong with it 🤔

Reply:
Oh, what. That was just GitHub rendering things in a weird way then.